src/Controls/samples/Controls.Sample/Maui.Controls.Sample.csproj

```diff
@@ -63,6 +63,7 @@
   <ItemGroup Condition=" '$(UseMaui)' == 'true' ">
     <PackageReference Include="Microsoft.Maui.Controls.Foldable" Version="$(MauiVersion)" />
     <PackageReference Include="Microsoft.Maui.Controls.Maps" Version="$(MauiVersion)" />
+    <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="$(MauiVersion)" />
```

Comment:
So, to get Compatibility in .NET 8, you add the NuGet?

Should we update one of the workload READMEs, giving an example?

Reply:
I added the breaking label, so we will make some docs